### ISSUE TITLE
problem ordering buttons are unlabeled

### ISSUE THREAD

**Issue by benetechMatt** [benetech/MathShare#1370]
buttons for moving moving problems up or down are not labeled. Read out as "button" to screen readers
video of issue https://youtu.be/YATjSEdzJBY?t=618
